.gitignore

```diff
@@ -2,3 +2,6 @@ tests/go.stripe/ff/customer_ffjson.go
 tests/goser/ff/goser_ffjson.go
 tests/types/ff/everything_ffjson.go
 tests/ff_ffjson.go
+
+# Vim temporaries
```

Comment:
I can remove this if you want. In my own repos, I always put a big `.gitignore` so other contributors won't accidentally commit stuff that then ends up in pull requests.